src/member/member.repository.ts

```diff
@@ -0,0 +1,49 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from '../prisma/prisma.service';
+import { UserGroup } from './interfaces/member.interface';
+
+@Injectable()
+export class MemberRepository {
+  constructor(private readonly prisma: PrismaService) {}
+
+  async findGroupMember(
+    groupId: number,
+    userId: number,
+  ): Promise<(UserGroup & { user?: { name: string } }) | null> {
```

Comment:
```suggestion
  ): Promise<(UserGroup & { user: User }) | null> {
```

유저가 특정 그룹에 가입하려면 계정이 있어야하지 않을까요??
그리고 include로 user를 가져오니 전체 유저를 응답하게되지 않나요??
프리즈마 타입에러때문에 optional로 지정하신거라면 prisma 스키마가 변경돼야겠구요
아래 findAllMembersByGroup도 같은 코멘트입니다

Reply:
아 제가 댓글을 잘못 달았네요. 전체유저라는 단어를 썻을 당시에 모든 유저 목록을 가져온다는 의미로 쓴건 아니고 해당 유저의 모든 속성을 말했던거예요
name만 가져오는것이 아닌 다른 필드도 가져옴